src/pages/index.js

```diff
@@ -153,18 +224,65 @@ function handleEscape(evt) {
     const modal = document.querySelector(".modal_opened");
     closeModal(modal);
   };
-
 };
 
 
 // Profile Submission Function
 function handleEditFormSubmit(evt) {
   evt.preventDefault();
-  profileName.textContent = editModalNameInput.value;
-  profileDescription.textContent = editModalDescriptionInput.value;
-  closeModal(editModal);
+  function makeRequest() {
+    return api.editUserInfo({
+      name: editModalNameInput.value,
+      about: editModalDescriptionInput.value
+    }).then((data) => {
+      profileName.textContent = data.name;
+      profileDescription.textContent = data.about;
```

Comment:
* Modals don't close after submission in `.then`

Please, close the modal here in `.then`